final/pkg/handle/handler.go

```diff
@@ -0,0 +1,74 @@
+package handle
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"strconv"
+
+	"github.com/NikhilSharmaWe/gomodule/final/pkg/database"
+)
+
+func CreateBook(w http.ResponseWriter, r *http.Request) {
+	w.Header().Set("Content-Type", "text/html; charset=utf-8")
+	fmt.Fprint(w, `
+	<form action="/" method="GET">
+        <input type="text" name="name" placeholder="name">
+        <input type="text" name="author" placeholder="author">
+        <input type="text" name="publication" placeholder="publication">
+        <input type="submit" name="submit" value="submit">
+    </form>
+    <form action="/getbooks" method="POST">
+        <input type="submit" name="submit" value="getbooks">
+    </form>
+    <form action="/book" method="GET">
+        <input type="text" name="bookId" placeholder="bookId">
+        <input type="submit" name="submit" value="getbookbyid">
+    </form>
+	`)
+	name := r.FormValue("name")
+	author := r.FormValue("author")
+	publication := r.FormValue("publication")
+	if name == "" || author == "" || publication == "" {
+		w.WriteHeader(http.StatusOK)
+		return
+	}
+	CreateBook := &database.Book{
+		Name:        name,
+		Author:      author,
+		Publication: publication,
+	}
+	CreateBook.CreateBook()
```

Comment:
The errors returned from db function will be handled here. We'll decide what message and status code, we need to send to the UI, according to the error.